packages/driver/test/cypress/integration/commands/querying_spec.coffee

```diff
@@ -883,6 +883,16 @@ describe "src/cy/commands/querying", ->
             .get("@getUsers").then (xhr) ->
               expect(xhr.url).to.include "/users"
 
+        it "handles dots in alias name", ->
```

Comment:
This test ends up throwing an error - when attempting to define an alias with a dot in its name for a route. I think if we will support dots in alias names, it should support it in routes as well as DOM els.

![Screen Shot 2019-05-07 at 12 43 02 PM](https://user-images.githubusercontent.com/1271364/57275997-2bab6880-70c6-11e9-8704-f3ee4cdd7439.png)

Reply:
I've added some code to handle the dot character in the route name. Are there any other kind of aliases that I should be aware?